source/bao_hyp/user_manual/porting.rst

```diff
@@ -0,0 +1,270 @@
+Porting
+=======
+
+The Bao Hypervisor only contains architecture-specific drivers (e.g., interrupt controllers or
+IOMMUS). Therefore porting the hypervisor essentially resumes to two simple operation: creating a
+platform description folder containing a platform definition source and header files which
+indicates, among others, the number of CPUs or the memory available. The exception is an UART driver
+used fore logging, so if a driver is not available for a UART on the target platform this might
```

Comment:
In my opinion, the first sentence doesn't contribute significantly. It looks like a bit disconnected from the remaining text.
```suggestion
Porting the hypervisor involves two basic steps: (i) creating a platform description folder with
source and header files that specify platform details, such as the number of CPUs or available
memory, and (ii) integrating a UART driver used for logging, so if a driver is not available for a
UART on the target platform this might require writing a custom driver. 
```